### ISSUE TITLE
Auto-add Title and Description using meta tags

### ISSUE THREAD

**Issue by xlogix** [fnplus/footsteps-extension#12]
When the extension is clicked it should be able to fetch the title and description using the meta tags of the website.

**Comment by Swati0604** [comment]
I want to work on this issue @xlogix

**Comment by xlogix** [comment]
Assigned!

**Comment by PritiShaw** [comment]
@xlogix  Is this issue available?

**Comment by xlogix** [comment]
Yes, it is. Assigned!

**Comment by PritiShaw** [comment]
![image](https://user-images.githubusercontent.com/34039705/80291369-912ab600-876a-11ea-9ac2-80ff8e624028.png)

Am I missing some dependency to install? I have followed readme.

**Comment by xlogix** [comment]
No, it's because we aren't importing the library the right way. You're welcome to raise an issue and fix it :)
